controllers/topics.controllers.js

```diff
@@ -0,0 +1,12 @@
+const{selectTopics} = require('../models/topics.models')
+
+exports.getTopics = (req, res, next) => {
+    console.log('in controller')
```

Comment:
Please remove console logs before pushing code. You can add them back in again if you need them for the next task.